docker/Dockerfile.docs

```diff
@@ -1,14 +1,15 @@
-FROM python:2.7.14-slim
+FROM python:2.7.15-slim
```

Comment:
Oops--we want to use the same image as the rest of things, so this should get downgraded to 2.7.14.